tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -405,7 +405,7 @@ public function testConnectionIsClosed()
     {
         $this->_conn->close();
 
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
+        //$this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
 
         $this->_conn->quoteIdentifier('Bug');
```

Comment:
IMO, this should still throw an exception if the `$platform` is null (connection is re-opened, platform is re-detected, platform is `null` due to failed automatic detection)

The exception type may change.